package.py

```diff
@@ -43,6 +45,15 @@ def main():
               'Default (from platform.architecture()): %(default)s'))
     args = parser.parse_args()
 
+    shutil.copyfile('build/src/out/Default/mini_installer.exe',
+        'build/ungoogled-chromium_{}-{}.{}_installer.exe'.format(
+            get_chromium_version(), _get_release_revision(), _get_packaging_revision()))
+
+    # We need to remove these files, or they'll end up in the zip files that will be generated.
+    os.remove('build/src/out/Default/mini_installer.exe')
+    os.remove('build/src/out/Default/mini_installer_exe_version.rc')
+    os.remove('build/src/out/Default/setup.exe')
```

Comment:
What does `setup.exe` do?

Reply:
That exe file adds Chromium to the list of programs in the start menu, the "Uninstall programs" menu in the Control Panel, and the "Apps & Features" menu in the settings, as far as I can see.

Reply:
Interesting, I'm assuming `mini_installer.exe` also does that?

Reply:
`mini_installer.exe` does that too.